src/index.js

```diff
@@ -81,21 +81,18 @@ function configureInterceptors(instance, url, { username, password } = {}) {
     return config;
   })
 
-  instance.interceptors.response.use(res => {
-    return res.data;
-  }, err => {
-    if (!(err instanceof Error) && typeof err === "object") {
-      return Promise.reject(errorify(err));
-    }
-    return Promise.reject(err);
-  });
+  instance.interceptors.response.use(
+    res => res.data,
+    err => Promise.reject(errorify(err))
+  );
 
   return instance;
 }
 
-function errorify(res) {
-  return new HttpError(res.status, res.statusText, {
-    response: res.data,
-    message: res.data.message
+function errorify(err) {
+  if (typeof err !== 'object') return new HttpError(500);
```

Comment:
I think we should have passed the error message in still or it will be hard to debug.